server/model/spans.go

```diff
@@ -9,6 +9,18 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+type TracetestMetadataField string
+
+const (
+	TracetestMetadataFieldStartTime TracetestMetadataField = "tracetest.span.start_time"
+	TracetestMetadataFieldEndTime   TracetestMetadataField = "tracetest.span.end_time"
+	TracetestMetadataFieldDuration  TracetestMetadataField = "tracetest.span.duration"
+	TracetestMetadataFieldType      TracetestMetadataField = "tracetest.span.type"
+	TracetestMetadataFieldName      TracetestMetadataField = "tracetest.span.name"
+	TracetestMetadataFieldParentID  TracetestMetadataField = "tracetest.span.parent_uid"
```

Comment:
is this field renamed to `parent_Uid` intentionally or is it a typo?
```suggestion
	TracetestMetadataFieldParentID  TracetestMetadataField = "tracetest.span.parent_id"
```

Reply:
This was definitely a fat finger